xrpl4j-integration-tests/src/test/java/org/xrpl/xrpl4j/tests/AmmIT.java

```diff
@@ -288,6 +293,562 @@ void depositAndWithdraw() throws JsonRpcClientErrorException, JsonProcessingExce
+        .account(traderKeyPair.publicKey().deriveAddress())
+        .fee(FeeUtils.computeNetworkFees(feeResult).recommendedFee())
+        .sequence(traderAccount.accountData().sequence())
+        .signingPublicKey(traderKeyPair.publicKey())
+        .limitAmount(IssuedCurrencyAmount.builder()
+            .currency(testCurrency)
+            .issuer(issuerKeyPair.publicKey().deriveAddress())
+            .value("1000")
+            .build()
+        )
+        .flags(TrustSetFlags.builder().tfClearNoRipple().build())
+        .build();
+
+    SingleSignedTransaction<TrustSet> signedTrustline =
+        signatureService.sign(traderKeyPair.privateKey(), trustSet);
+    SubmitResult<TrustSet> trustlineSubmitResult = xrplClient.submit(signedTrustline);
+    assertThat(trustlineSubmitResult.engineResult()).isEqualTo(TransactionResultCodes.TES_SUCCESS);
+
+    // create Trustline from issuer to trader for ZFT
+    TrustSet trustSet2 = TrustSet.builder()
+        .account(traderKeyPair.publicKey().deriveAddress())
+        .fee(FeeUtils.computeNetworkFees(feeResult).recommendedFee())
+        .sequence(traderAccount.accountData().sequence().plus(UnsignedInteger.ONE))
+        .signingPublicKey(traderKeyPair.publicKey())
+        .limitAmount(
+            IssuedCurrencyAmount.builder()
+                .currency(xrpl4jCoin)
+                .issuer(issuerKeyPair.publicKey().deriveAddress())
+                .value("1000000")
+                .build()
+        )
+        .flags(TrustSetFlags.builder().tfClearNoRipple().build())
+        .build();
+
+    SingleSignedTransaction<TrustSet> signedTrustline2 =
+        signatureService.sign(traderKeyPair.privateKey(), trustSet2);
+    SubmitResult<TrustSet> trustlineSubmitResult2 = xrplClient.submit(signedTrustline2);
+    assertThat(trustlineSubmitResult2.engineResult()).isEqualTo(TransactionResultCodes.TES_SUCCESS);
+
+    // send payment of ZFT to trader wallet from issuer
+    Payment paymentZFT = Payment.builder()
+        .account(issuerKeyPair.publicKey().deriveAddress())
+        .destination(traderKeyPair.publicKey().deriveAddress())
+        .amount(IssuedCurrencyAmount.builder()
+            .currency(xrpl4jCoin)
+            .issuer(issuerKeyPair.publicKey().deriveAddress())
+            .value("500")
+            .build())
+        .fee(FeeUtils.computeNetworkFees(feeResult).recommendedFee())
+        .sequence(issuerAccount.accountData().sequence().plus(UnsignedInteger.valueOf(2)))
+        .signingPublicKey(issuerKeyPair.publicKey())
+        .build();
+
+    SingleSignedTransaction<Payment> signedPaymentZFT =
+        signatureService.sign(issuerKeyPair.privateKey(), paymentZFT);
+    SubmitResult<Payment> paymentZFTSubmitResult = xrplClient.submit(signedPaymentZFT);
+    assertThat(paymentZFTSubmitResult.engineResult()).isEqualTo(TransactionResultCodes.TES_SUCCESS);
+
+    // send payment of TST to trader wallet from issuer
+    Payment paymentTST = Payment.builder()
```

Comment:
This naming convention breaks our Checkstyle plugin. Need to replace all instances of `...TFT` with `Tft`. For example:

```suggestion
    Payment paymentTft = Payment.builder()
```

Reply:
i've updated this along with all the other checkstyle issues